README.md

```diff
@@ -94,114 +94,130 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
 ```
-$ <snapBinPath>/snapd --plugin-trust 0 --log-level 1
+$ snapd --plugin-trust 0 --log-level 1
 ```
-This will bring up a snap agent without requiring plugin signing and set the logging level to debug. snap's REST API will be listening on port 8181. To learn more about the snap agent and how to use it look at [SNAPD.md](docs/SNAPD.md) and/or run `<snapBinPath>/snapd -h`.
+This will bring up a snap agent without requiring plugin signing (trust-level 0) and set the logging level to debug (log level 1). Snap's REST API will be listening on port 8181. To learn more about the snap agent and how to use it look at [SNAPD.md](docs/SNAPD.md) and/or run `snapd -h`.
 
-### Running snap in tribe (cluster) mode
+Snap can also be run in a clustered mode called `tribe`. Checkout the [tribe documentation](docs/TRIBE.md) for more info.
 
-Start the first node:
+### Load Plugins
+Snap gets its power from the use of plugins. The [Plugin Catalog](#plugin-catalog) is a collection of all known plugins for Snap with links to the binaries. You can download individual plugins or pull down a package of starter plugins for your operating system under [GitHub Releases](https://github.com/intelsdi-x/snap/releases). This isn't the comprehensive list of plugins, but they will help you get started.
 
+After downloading the package (example is on Linux):
 ```
-$ <snapBinPath>/snapd --tribe
+$ tar -xvf snap-plugins-v0.13.0-beta-linux-amd64.tar.gz
+$ mkdir -p ~/snap/plugins/
+$ mv snap-v0.13.0-beta/plugin/* ~/snap/plugins/
+$ rm -rf snap-v0.13.0-beta
 ```
 
-All other nodes who join will need to select any existing member of the cluster.
+Next, load the plugins. This can be achieved through the REST API directly or by using the helper command `snapctl`.
 
+Using the API directly with cURL:
 ```
-$ <snapBinPath>/snapd --tribe --tribe-seed <ip or name of another tribe member>
+$ cd ~/snap/plugins/
+$ curl -X POST -F plugin=@snap-collector-mock1 http://localhost:8181/v1/plugins
+$ curl -X POST -F plugin=@snap-processor-passthru http://localhost:8181/v1/plugins
+$ curl -X POST -F plugin=@snap-publisher-file http://localhost:8181/v1/plugins
 ```
 
-Checkout the [tribe doc](docs/TRIBE.md) for more info.
-
-## Load Plugins
-snap gets its power from the use of plugins. The [plugin catalog](#plugin-catalog) is a collection of all known plugins for snap.
-
-Next, open a new shell and lets load a few of the demo plugins.  You can do this via cURL, or `snapctl`, snap's CLI.
-
-Using cURL
-```sh
-$ cd $SNAP_PATH
-$ curl -X POST -F plugin=@plugin/snap-collector-mock1 http://localhost:8181/v1/plugins
-$ curl -X POST -F plugin=@plugin/snap-processor-passthru http://localhost:8181/v1/plugins
-$ curl -X POST -F plugin=@plugin/snap-publisher-file http://localhost:8181/v1/plugins
-```
-
-Or:
-
-Using `snapctl` (snap CLI)
+Every interaction with `snapd` can be done through the REST API. To continue using it, see the [API Documentation](docs/REST_API.md). We will continue on using `snapctl`:
```

Comment:
Not sure if "To continue using it" is the right thing here. Think it would be more of a to see what else you can do with the REST API type of thing.